backend/src/serverless/integrations/services/integrationProcessor.ts

```diff
@@ -246,7 +271,22 @@ export class IntegrationProcessor extends LoggingBase {
               { stream: stream.value, remainingStreams: streams.length },
               `Processing stream.`,
             )
-            const processStreamResult = await intService.processStream(stream, stepContext)
+            let processStreamResult
+            try {
+              processStreamResult = await intService.processStream(stream, stepContext)
+            } catch (err) {
+              if (err.rateLimitResetSeconds) {
+                delay = err.RateLimitResetSeconds + 5
```

Comment:
nit: can we use small r in RateLimitReset... here

Reply:
It's actually a bug that you have found here :) thank you